.NET/Microsoft.Recognizers.Text.DateTime/Utilities/AgoLaterUtil.cs

```diff
@@ -225,106 +229,94 @@ public static class AgoLaterUtil
                 ret.Mod = Constants.LESS_THAN_MOD;
             }
 
-            if (MatchingUtil.ContainsAgoLaterIndex(afterStr, utilityConfiguration.AgoRegex))
+            int swift = 0;
+            bool isMatch = false, isLater = false;
+            string dayStr = null;
+            var agoLaterRegexes = new List<(Regex, string)>

```

Comment:
Same. Please also declare the special string as Constants.

Reply:
done